C++/Toolbox/Option/Option.h

```diff
@@ -0,0 +1,127 @@
+#ifndef _OPTION_H_
```

Comment:
We have also switched to use #pragma once

Reply:
I've swapped out the guards for pragma once